std/mutex.zig

```diff
@@ -21,7 +36,28 @@ pub const Mutex = struct {
     }
 
     pub fn acquire(self: *Mutex) Held {
-        while (@atomicRmw(u8, &self.lock, builtin.AtomicRmwOp.Xchg, 1, AtomicOrder.SeqCst) != 0) {}
+        var c: u32 = undefined;
+        // This need not be strong because of the loop that follows.
+        // TODO implement mutex3 from https://www.akkadia.org/drepper/futex.pdf in x86 assembly.
```

Comment:
Is that really better? Can we have an issue to discuss it rather than this TODO comment?